pontoon/base/models/entity.py

```diff
@@ -845,9 +846,13 @@ def for_project_locale(
         if search:
```

Comment:
I'd also add a TODO here with a bullet point for each of the (currently 3) search options that are disabled ATM and instructions on how to enable them. That would also supersede the TODO in lines 867-868.